server/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -250,6 +252,7 @@ private void active() throws StartupException {
       logger.error("Meet error while starting up.", e);
       throw new StartupException("Error in activating IoTDB DataNode.");
     }
+    activated = true;
```

Comment:
It seems that activated cannot turn false when deactivate the datanode.

Reply:
Fixed, thank you.